crates/bevy_math/src/bounding/bounded2d.rs

```diff
@@ -0,0 +1,284 @@
+use super::BoundingVolume;
+use crate::prelude::Vec2;
+
+/// A trait with methods that return 2d bounded volumes for a shape
+pub trait Bounded2d {
+    /// Get an axis-aligned bounding box for the shape with the given translation and rotation
```

Comment:
We really need to specify what the f32 means for rotation here. Radians? In what direction? Starting where?

Reply:
(I'd personally like a `Radians` type, but I know that will be more controversial so we shouldn't do it here).